python_src/src/lamp_py/tableau/jobs/rt_rail.py

```diff
@@ -71,6 +73,16 @@ def __init__(self) -> None:
             "   vehicle_events prev_ve"
             " ON "
             "   ve.pm_event_id = prev_ve.next_trip_stop_pm_event_id"
+            " LEFT JOIN "
+            "   static_stops ss"
```

Comment:
just noticing this, but is the `AS` keyword unnecessary?

Reply:
It's generally considered to be optional: https://www.postgresql.org/docs/current/queries-table-expressions.html#QUERIES-TABLE-ALIASES